### PR TITLE
Highlight HCP Waypoint under /hcp landing pages

### PR DESCRIPTION
**Pull request by ashleemboyer**
## 🔗 Relevant links

- [Asana task](https://app.asana.com/0/1202097197789424/1203115842976152/f) 🎟️
- [Preview](https://dev-portal-git-ambadd-hcp-waypoint-highlights-hashicorp.vercel.app/hcp)

## 🗒️ What

- Adds HCP Waypoint `IconCardLink` in the Getting Started callout card on `/hcp`
- Adds HCP Waypoint `IconCardLink` to the top of `/hcp/docs`
- Adds HCP Waypoint section to sidebars under `/hcp/tutorials`

## 🤷 Why

- The HCP Waypoint content is new and hadn't been highlighted yet. 😊

### HCP landing page

- [ ] Go to [/hcp](https://dev-portal-git-ambadd-hcp-waypoint-highlights-hashicorp.vercel.app/hcp)
- [ ] Make sure there is an HCP Waypoint `IconCardLink` in the Getting Started callout card
- [ ] Make sure the card links to the correct page

image

### HCP docs landing page

- [ ] Go to [/hcp/docs](https://dev-portal-git-ambadd-hcp-waypoint-highlights-hashicorp.vercel.app/hcp/docs)
- [ ] Make sure there is an HCP Waypoint `IconCardLink` at the top of the page
- [ ] Make sure the card links to the correct page

image

### HCP tutorials landing page

- [ ] Go to [/hcp/tutorials](https://dev-portal-git-ambadd-hcp-waypoint-highlights-hashicorp.vercel.app/hcp/tutorials)
- [ ] Make sure there is an HCP Waypoint section towards the bottom of the sidebar
- [ ] Make sure the text of the one item in the section is correct
- [ ] Make sure the link of the one item in the section is correct

image

## 💭 Anything else?

Questions:

- Should the new sidebar item have a "Beta" badge since HCP Waypoint's docs show one?
- Should the two sidebar items under Boundary still have "Beta" badges? (We can resolve in another PR, or this one)
- Are the links correct for the "Getting Started" callout card and the new sidebar item?

### CODE REVIEW

File: src/content/hcp/product-landing.json
Comment (kendallstrautman) [inline src/content/hcp/product-landing.json:40]: I think maybe we just don't include waypoint here since there isn't a tru getting started collection? I'll defer to @topfunky though.

File: src/content/hcp/tutorials-sidebar.json
Comment (kendallstrautman) [inline src/content/hcp/tutorials-sidebar.json:45]: Good point in your questions, I think we should remove the `beta` tags from Boundary. Could totally be in a follow up PR though.

Comment (kendallstrautman) [reply src/content/hcp/tutorials-sidebar.json:45]: there's also 'BETA' in a card section title on the[ tutorials landing page.  ](https://dev-portal-git-ambadd-hcp-waypoint-highlights-hashicorp.vercel.app/hcp/tutorials)

**Review by kendallstrautman — APPROVED** [review]
Looks good on my end! But we should get clarify on the getting started link from @topfunky before merging 🌟

File: src/content/hcp/product-landing.json
Comment (tunzor) [reply src/content/hcp/product-landing.json:40]: I'd like to leave it in as it highlights that HCP Waypoint is available if someone is browsing for HCP services. 

Unfortunately it's the only tutorial we have so not a true getting started though you're right 😅

**Review by tunzor — APPROVED** [review]
👍

**Review by topfunky — APPROVED** [review]
👍🏻

**Pull request merged**
